Makefile

```diff
@@ -34,6 +34,7 @@ all_generated_files := \
 .PHONY: install-build-deps
 install-build-deps: ## Install dependencies (packages and tools)
 	./install-build-deps.sh
+	pip3 install -r userdocs/requirements.txt
```

Comment:
@dholbach  We might need to tweak the scripts that make sure the docker image is up to date so that it changes the manifest whenever this file has changed

Reply:
Not sure they will be needed on the Docker images. Netlify makes sure to install these things: https://docs.netlify.com/configure-builds/common-configurations/#mkdocs

Reply:
But yeah, if you need it in a different place, just let me know and I'll take a look.